ext/nnstreamer/tensor_filter/tensor_filter_pytorch.cc

```diff
@@ -489,7 +489,8 @@ TorchCore::invoke (const GstTensorMemory *input, GstTensorMemory *output)
     g_assert (outputTensorMeta.num_tensors == output_list.size ());
     int idx = 0;
     for (auto &ivalue_element : output_list) {
-      if (processIValue (ivalue_element, &output[idx++])) {
+      ++idx;
```

Comment:
Q: index started from 1? plz check this line.

Reply:
My mistake. I moved the line, thanks.